proc/groupby.go

```diff
@@ -303,67 +342,71 @@ func (g *GroupByAggregator) recordsForTable(table map[string]*GroupByRow) []*zso
 	}
 	sort.Strings(keys)
```

Comment:
Unrelated to this PR but what's the point of this?

Reply:
I think to have stable results, e.g., for test baselines, but I'm not sure.